src/Layout/Presentation/DistrictOverview/DistrictOverview.tsx

```diff
@@ -40,6 +42,19 @@ export class DistrictOverview extends React.Component<DistrictOverviewProps, {}>
                 </span>
                 <br />
                 <br />
+                <span>
+                    {"Det mest utsatte sistemandatet var i "}
```

Comment:
This turned out to be incredibly messy and unreadable, so I decided against it (it would be much more tedious to ensure everything worked properly as well, due to layering JSX-compatible expressions).